tests/_ast/test_sql_visitor.py

```diff
@@ -593,7 +601,7 @@ def test_find_sql_refs_with_lateral_join() -> None:
         SELECT * FROM employees,
         LATERAL (SELECT * FROM departments WHERE departments.id = employees.dept_id) dept;
         """
-        assert find_sql_refs(sql) == ["employees", "departments"]
+        assert find_sql_refs(sql) == ["departments", "employees"]
```

Comment:
i need to investigate more on this ordering. Nested tables are referenced first, I'm not sure what impact that has.